app/views/admin/find_in_admin_bookmarklet/_bookmarklet.erb

```diff
@@ -10,12 +10,14 @@ function id_from_url() {
 };
 function id_from_doc_page() {
   document_page_id = $('.document-page, [id^=detailed_guide_]');
-  if (document_page_id.length > 0) {
+  if (document_page_id == null) {
+    return false;
+  } else if (document_page_id.length > 0) {
     return document_page_id.attr('id').split('_').pop();
   }
 };
 function content_id_from_meta_tag() {
-  return $("meta[name='govuk:content-id']").attr('content');
+  return document.querySelector('meta[name="govuk:content-id"]').content;
```

Comment:
Just curious why this has been converted to vanilla JS instead of jQuery - it's okay to use jQuery, there's loads of it in the rest of this file.